build/args/all.gn

```diff
@@ -10,7 +10,6 @@ enable_extensions = false
 proprietary_codecs = true
 ffmpeg_branding = "Chrome"
 
-# TODO(jeremy): OSR is broken in M68. Remove this once it's fixed.
-enable_osr = false
+enable_osr = true
```

Comment:
The flag should be removed, actual build flags are under `buildflags/buildflags.gni`, please use them for disabling/re-enabling features in the future. Thanks!